fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -109,6 +125,42 @@ TEST_P(Int32Arithmetic, UsingLoadParam) {
     ASSERT_EQ(param.oracle(param.lhs, param.rhs), entry_point(param.lhs, param.rhs));
 }
 
+TEST_P(UInt32Arithmetic, UsingConst) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120, "(method return=Int32 (block (ireturn (%s (iconst %d) (iconst %d)) )))", param.opcode.c_str(), param.lhs, param.rhs);
+    auto trees = parseString(inputTrees);
+
+    ASSERT_NOTNULL(trees);
+
+    Tril::DefaultCompiler compiler{trees};
+
+    ASSERT_EQ(0, compiler.compile()) << "Compilation failed unexpectedly\n" << "Input trees: " << inputTrees;
+
+    auto entry_point = compiler.getEntryPoint<uint32_t (*)(void)>();
+    volatile auto exp = param.oracle(param.lhs, param.rhs);
+    volatile auto act = entry_point();
+    ASSERT_EQ(exp, act);
+}
+
+TEST_P(UInt32Arithmetic, UsingLoadParam) {
+    auto param = TRTest::to_struct(GetParam());
+
+    char inputTrees[120] = {0};
+    std::snprintf(inputTrees, 120, "(method return=Int32 args=[Int32, Int32] (block (ireturn (%s (iload parm=0) (iload parm=1)) )))", param.opcode.c_str());
```

Comment:
I suggest we format these strings so they're easier to read. See example in [1]. Similarly elsewhere in the PR.

[1] https://github.com/eclipse/omr/blob/3e2bef8e9e0840ef3ddb61abc3f2bf5a8b4e0bab/fvtest/compilertriltest/IfxcmpgeReductionTest.cpp#L98-L114

Reply:
Sorry, I must've somehow missed the notification for this comment. Fixed in 4d37a00315f4bab7ceaae1c567c91baafed7afb5. I also went ahead and formatted the rest of the strings used as templates for trees in `ArithmeticTest.cpp`.